### PR TITLE
Create RELEASE.md

### PR DESCRIPTION
**Pull request by kSee04**
This is part of https://github.com/kabanero-io/kabanero-website/issues/244

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Docs Release
+Docs follow the release process for Kabanero as a whole - Kabanero uses [semver](https://semver.org/) versioning.
+
+Here is an example flow of the Kabanero release using `0.2.0` as the next release example:
+
+**Assumption**: When a new release of Kabanero as a whole is ready the next version number will always be a **Major** or **Minor** increment (never a **Patch** increment).
```
Comment (garypicher) [inline RELEASE.md]: This is basically correct.  We will cut a release candidate first (e.g. 0.2.0-rc1) and then remove the rc when it's verified.  The major or minor version will increment from the last release, and the patch version will be 0.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Docs repo needs a hot fix for `0.2.0` example
+
+1. If after a release, A piece of Kabanero.io (cli, pipelines, operator, docs, etc...) needs to do a small update or a quick bug fix or something (and should be deployed before the next scheduled release, `0.3.0`, in this example) maintainers would:
+1. create a `release-0.2` branch based off the `0.2.0` release. 
+1. Put the changes in the `release-0.2` branch and `master` branch, if applicable (see Note below).
+1. Cut a new release based off the `release-0.2` branch and increment the patch number so it becomes `0.2.1`
+1. Publish release `0.2.1`.
+
+**Note**: During this whole hot fix situation the `master` branch will always be going towards the next release so new stuff in master will always be `0.3.0` related.
```
Comment (garypicher) [inline RELEASE.md]: Are we talking about patch fixes to the docs repo itself, or doc updates that correspond to a patch release of kabanero?

The answer, I think, is that once we have a "release of kabanero" (e.g. 0.2.0) and a corresponding release of docs with the same version number (again, 0.2.0), we can patch *either* of those releases, but the patch versions don't need to match.

For example, let's say we find a piece of doc that is just flat out wrong, but the function itself works okay.  We'd want to create a 0.2.1 release of the *docs* repo, but we don't actually need a new release of kabanero.  Alternately, we might need to patch kabanero and create a 0.2.1 release of kabanero, but if the docs are still correct, we don't need a new release of docs.

So basically, for a given release, the patch version will increment for both the product code and for the docs, but they won't necessarily match.

Comment (alohr51) [reply RELEASE.md]: yea, this is talking about patches for the docs repo only (if a doc is wrong after we release, but not necessarily Kabanero itself). I agree the patch versions don't have to match, because if **only** the docs are wrong Kabanero will stay at 0.2.0 and docs can be bumped to 0.2.1 and published to fix the mistake and vice versa.
